### PR TITLE
GNOM-2113--bugfix2: Changing the answer to the "lab prep" question updates the list of experiment types.

### PR DESCRIPTION
**Pull request by matthewbjorge**
[Better quality diff](https://github.com/hci-gnomex/gnomex/pull/36/files?w=0)

You'll see I moved code from the second half of the init() method into a new reloadApplications() method so it could be called from another method as well.

### CODE REVIEW

**Comment by tmaness** [comment]
Looks good to me.

**Comment by jdewell** [comment]
Looks good. Thanks for looking into this.

**Comment by tcourdy** [comment]
Looks good.  I will merge.

File: flex/views/experiment/TabSeqSetupView.mxml [hunk 1]
```diff
@@ -105,64 +88,59 @@
...
+			var codeRequestCategory:String = parentDocument.getRequestCategory().@codeRequestCategory;
+			if (codeRequestCategory != previousCodeRequestCategory) {
+				previousCodeRequestCategory = codeRequestCategory;
+				reloadApplications();
+			}
+		}
 
-				parentDocument.filteredAppList.refresh();
+		private function reloadApplications():void {
+			var done:Boolean = parentDocument.filteredAppList.refresh();
```
Comment (tcourdy) [inline flex/views/experiment/TabSeqSetupView.mxml]: You initialize "done" here but it doesn't appear that you use the value.  I just want to make sure that you didn't need to use it somewhere before I merge it.  Thanks.

Comment (matthewbjorge) [inline flex/views/experiment/TabSeqSetupView.mxml]: Ah, good catch. I don't need the value of "done" - I'd been testing). The line should just be:
      parentDocument.filteredAppList.refresh();

Comment (tcourdy) [inline flex/views/experiment/TabSeqSetupView.mxml]: Okay cool just wanted to be sure.  I'll wait for you to update that line and then I will merge it with master.  Thanks Matt.

**Comment by tcourdy** [comment]
I went ahead and just pushed the change myself.  I will merge this now.

**Pull request merged**